src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.*;
```

Comment:
Maybe you could import just the ArrayList instead of everything from the Java util library?